PokeApiNet.Tests/Cache/MockCacheExpirationOptionsSource.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace PokeApiNet.Tests.Cache
+{
+    internal sealed class MockCacheExpirationOptionsSource : IObservable<CacheExpirationOptions>
```

Comment:
I would prefer if we used Moq to stub out this implementation for use in our tests instead of having an additional class with logic in it. I tested out using Moq and it still works the same and without issue.

Reply:
I added the class in a copy-paste manner just to get the project to build and be green. Mind if I use `NSubstitute` instead of `Moq`? :sweat_smile: